readme.md

```diff
@@ -127,7 +127,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [MacTerm](https://www.macterm.net/) - Powerful replacement for macOS Terminal.
 * [Mosh](https://github.com/mobile-shell/mosh) - Remote terminal application that allows roaming and supports intermittent connectivity.
 * [PuTTY](https://www.putty.org/) - SSH and telnet client, developed originally by Simon Tatham for the Windows platform.
-* [Terminator](https://gnometerminator.blogspot.com/p/introduction.html) - The robot future of terminals.
+* [Terminator](https://github.com/gnome-terminator/terminator) - The robot future of terminals.
```

Comment:
Hey, thanks for your contribution! Could you also update the description to reflect the one on the new GitHub repo?